dcplib/component_entities/ingest_entities.py

```diff
@@ -248,6 +259,10 @@ def processes(self):
         return [Process(process_data, ingest_api_agent=self.api) for process_data in
                 self.api.get_all(self.data['_links']['processes']['href'], 'processes', page_size=20)]
 
+    def submission_errors(self):
+        return [SubmissionError(error_data, ingest_api_agent=self.api) for error_data in
+                self.api.get_all(self.data['_links']['submissionEnvelopeErrors']['href'], 'submissionErrors', page_size=20)]
```

Comment:
Will the hardcoded page size mean that you only see the first 20 errors?

Reply:
This is just the page size. api.get_all() pages through and returns all the results